mobile/v1/redshift/sql-runner/sql/standard/01-base/XX-destroy/XX-destroy-base.sql

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Nit: Should destroy events_staged table here

Reply:
Good catch. Updated